apps/bfd-model/bfd-model-rif/src/main/java/gov/cms/bfd/model/rif/RifFileType.java

```diff
@@ -1,32 +1,35 @@
 package gov.cms.bfd.model.rif;
 
+import java.util.Optional;
+
 /** Enumerates the various types of RIF files. */
 public enum RifFileType {
-  BENEFICIARY(BeneficiaryColumn.class, BeneficiaryColumn.BENE_ID),
+  BENEFICIARY(BeneficiaryColumn.class, Optional.of(BeneficiaryColumn.BENE_ID)),
 
-  BENEFICIARY_HISTORY(BeneficiaryHistoryColumn.class, null),
+  BENEFICIARY_HISTORY(BeneficiaryHistoryColumn.class, Optional.empty()),
 
   MEDICARE_BENEFICIARY_ID_HISTORY(
-      MedicareBeneficiaryIdHistoryColumn.class, MedicareBeneficiaryIdHistoryColumn.BENE_MBI_ID),
+      MedicareBeneficiaryIdHistoryColumn.class,
+      Optional.of(MedicareBeneficiaryIdHistoryColumn.BENE_MBI_ID)),
 
-  CARRIER(CarrierClaimColumn.class, CarrierClaimColumn.CLM_ID),
+  CARRIER(CarrierClaimColumn.class, Optional.of(CarrierClaimColumn.CLM_ID)),
 
-  DME(DMEClaimColumn.class, DMEClaimColumn.CLM_ID),
+  DME(DMEClaimColumn.class, Optional.of(DMEClaimColumn.CLM_ID)),
 
-  HHA(HHAClaimColumn.class, HHAClaimColumn.CLM_ID),
+  HHA(HHAClaimColumn.class, Optional.of(HHAClaimColumn.CLM_ID)),
 
-  HOSPICE(HospiceClaimColumn.class, HospiceClaimColumn.CLM_ID),
+  HOSPICE(HospiceClaimColumn.class, Optional.of(HospiceClaimColumn.CLM_ID)),
 
-  INPATIENT(InpatientClaimColumn.class, InpatientClaimColumn.CLM_ID),
+  INPATIENT(InpatientClaimColumn.class, Optional.of(InpatientClaimColumn.CLM_ID)),
 
-  OUTPATIENT(OutpatientClaimColumn.class, OutpatientClaimColumn.CLM_ID),
+  OUTPATIENT(OutpatientClaimColumn.class, Optional.of(OutpatientClaimColumn.CLM_ID)),
 
-  PDE(PartDEventColumn.class, PartDEventColumn.PDE_ID),
+  PDE(PartDEventColumn.class, Optional.of(PartDEventColumn.PDE_ID)),
 
-  SNF(SNFClaimColumn.class, SNFClaimColumn.CLM_ID);
+  SNF(SNFClaimColumn.class, Optional.of(SNFClaimColumn.CLM_ID));
 
   private final Class<Enum<?>> columnEnum;
-  private final Enum<?> idColumn;
+  private final Optional<Enum<?>> idColumn;
```

Comment:
there is really no need to do this -- in the get function below you can simply:

`return Optional.ofNullable(idColumn);`